cumulus/client/cli/src/lib.rs

```diff
@@ -296,7 +296,14 @@ pub struct RunCmd {
 	#[arg(long, conflicts_with = "validator")]
 	pub collator: bool,
 
-	/// EXPERIMENTAL: Specify an URL to a relay chain full node to communicate with.
+	/// Creates a node that uses fewer resources by retrieving relay chain data from a remote node.
+	/// The provided URLs should point to RPC endpoints of the relay chain.
```

Comment:
```suggestion
	/// Creates a node that uses fewer resources by retrieving relay chain data from a remote node.
	///
	/// The provided URLs should point to RPC endpoints of the relay chain.
```

Not long ago I've made [this](https://github.com/paritytech/polkadot-sdk/pull/2073) PR where I make the CLI output readable.

With the `-h` option the "short" help is shown.

It is best if this short output is a "one liner".

Reply:
Maybe the fact that "uses fewer resources" can be part of the description below (i.e. not part of the short one liner).